crates/client/block_import/src/verify_apply/classes.rs

```diff
@@ -47,9 +47,52 @@ pub fn class_trie_root(
 #[cfg(test)]
 mod tests {
     use super::*;
+    use mp_chain_config::ChainConfig;
+    use std::sync::Arc;
 
     #[test]
     fn test_contract_class_hash_version() {
         assert_eq!(CONTRACT_CLASS_HASH_VERSION, Felt::from_bytes_be_slice(b"CONTRACT_CLASS_LEAF_V0"));
     }
+
+    #[test]
+    fn test_class_trie_root() {
```

Comment:
it would be good to add legacy classes with old_declared_classes for testing purposes

Reply:
here we are doing just the functional testing, and no actual value has been used

Reply:
`ReplacedClassItem` should also be able to contain a legacy class imo